exchanges/lbank/lbank_wrapper.go

```diff
@@ -0,0 +1,546 @@
+	tickerNew, err := ticker.GetTicker(l.GetName(), p, assetType)
+	if err != nil {
+		return l.UpdateTicker(p, assetType)
+	}
+	return tickerNew, nil
+}
+
+// GetOrderbookEx returns orderbook base on the currency pair
+func (l *Lbank) GetOrderbookEx(currency currency.Pair, assetType string) (orderbook.Base, error) {
+	ob, err := orderbook.Get(l.GetName(), currency, assetType)
+	if err != nil {
+		return l.UpdateOrderbook(currency, assetType)
+	}
+	return ob, nil
+}
+
+// UpdateOrderbook updates and returns the orderbook for a currency pair
+func (l *Lbank) UpdateOrderbook(p currency.Pair, assetType string) (orderbook.Base, error) {
+	var orderBook orderbook.Base
+	a, err := l.GetMarketDepths(p.String(), "60", "1")
+	if err != nil {
+		return orderBook, err
+	}
+	for i := range a.Asks {
+		orderBook.Asks = append(orderBook.Asks, orderbook.Item{
+			Price:  a.Asks[i][0],
+			Amount: a.Asks[i][1]})
+	}
+	for i := range a.Bids {
+		orderBook.Bids = append(orderBook.Bids, orderbook.Item{
+			Price:  a.Bids[i][0],
+			Amount: a.Bids[i][1]})
+	}
+	orderBook.Pair = p
+	orderBook.ExchangeName = l.GetName()
+	orderBook.AssetType = assetType
+	err = orderBook.Process()
+	if err != nil {
+		return orderBook, err
+	}
+
+	return orderbook.Get(l.Name, p, assetType)
+}
+
+// GetAccountInfo retrieves balances for all enabled currencies for the
+// Lbank exchange
+func (l *Lbank) GetAccountInfo() (exchange.AccountInfo, error) {
+
+	var info exchange.AccountInfo
+	data, err := l.GetUserInfo()
+	if err != nil {
+		return info, err
+	}
+	var account exchange.Account
+	for key, val := range data.Info.Asset {
+		c := currency.NewCode(key)
+		hold, ok := data.Info.Freeze[key]
+		if !ok {
+			return info, fmt.Errorf("hold data not found with %s", key)
+		}
+		totalVal, err := strconv.ParseFloat(val, 64)
+		if err != nil {
+			return info, err
+		}
+		totalHold, err := strconv.ParseFloat(hold, 64)
+		if err != nil {
+			return info, err
+		}
+		account.Currencies = append(account.Currencies,
+			exchange.AccountCurrencyInfo{CurrencyName: c,
+				TotalValue: totalVal,
+				Hold:       totalHold})
+	}
+
+	info.Accounts = append(info.Accounts, account)
+	info.Exchange = l.GetName()
+	return info, nil
+}
+
+// GetFundingHistory returns funding history, deposits and
+// withdrawals
+func (l *Lbank) GetFundingHistory() ([]exchange.FundHistory, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// GetExchangeHistory returns historic trade data since exchange opening.
+func (l *Lbank) GetExchangeHistory(p currency.Pair, assetType string) ([]exchange.TradeHistory, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// SubmitOrder submits a new order
+func (l *Lbank) SubmitOrder(p currency.Pair, side exchange.OrderSide, _ exchange.OrderType, amount, price float64, clientID string) (exchange.SubmitOrderResponse, error) {
+	var resp exchange.SubmitOrderResponse
+	if side != "BUY" && side != "SELL" {
```

Comment:
You can use our built-in OrderSide consts, "BUY" can be replaced with exchange.BuyOrderSide and "SELL" with exchange.SellOrderSide